.github/workflows/integration.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         id: extract-tests
         run: |
           # shellcheck disable=SC2046
-          echo "matrix={\"test\":$(jq -c -n '$ARGS.positional' --args $(grep '^func Test' $(find integration -name '*_test.go') | awk '{print $2}' | cut -d'(' -f1))}" >> "$GITHUB_OUTPUT"
+          echo "matrix={\"test\":$(jq -c -n '$ARGS.positional' --args $(grep -r -l '^//go:build integration' integration backend/controller/cronjobs | xargs grep '^func Test' | awk '{print $2}' | cut -d'(' -f1))}" >> "$GITHUB_OUTPUT"
```

Comment:
It will find them but it still won't execute them because line 45 needs to include `backend/controller/cronjobs`.

Reply:
Love the solution though!

Reply:
Ah I missed that